responsive.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">

```

Comment:
I'm curious to know what happens if we don't include this?
`content="width=device-width, initial-scale=1.0"`